src/components/views/MovieCard.vue

```diff
@@ -54,22 +54,29 @@ export default defineComponent({
 }
 .card-2 {
     @media only screen and (min-width: 768px) and (max-width: 1024px) {
-        margin-right: 0;
+      margin-right: 0;
+    }
+    @media only screen and (max-width: 768px) {
+      margin: 0px;
     }
 }
 .card-3 {
     margin-right: 0;
 
     @media only screen and (min-width: 768px) and (max-width: 1024px) {
-        display: none;
+      display: none;
+    }
+
+    @media only screen and (max-width: 768px) {
```

Comment:
I highly recommend extracting the media queries to sass mixins - this way we only need to define the exact sizes once instead of copying/repeating them throughout our styles 💡

Reply:
Good point! I actually haven't seen any need to use mixins since I started to build apps based on components. But I admit that in case of media queries they do a brilliant job! Thanks for this suggestion!